scripts/globals/furniture_quests.lua

```diff
@@ -61,17 +61,19 @@ xi.furnitureQuests.onFurniturePlaced = function(player, item)
 
     -- Dont allow more than one reward from an item per player if not repeatable
     if
```

Comment:
This was cleaner to remove from the if - and instead just return when criteria was met